src/Nox.Reference.Abstractions/VatNumbers/VatValidationStatus.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Nox.Reference.Abstractions;
+
+public enum VatValidationStatus
+{
+    NotVerified = 1,
```

Comment:
Should we add "Unknown = 0" as default?

Reply:
I guess Unknown is the same as NotVerified ?

Reply:
I think UnVerified is clearer and less ambiguous. 

"This VAT info is unverified"

Is better than

"This VAT info is unknown"

Reply:
So, just leave as is. NotVerified )

Reply:
Set Unverified